pkgs/development/python-modules/nanoleaf/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, buildPythonPackage, fetchPypi, requests }: #, lxml, requests, tkinter }:
```

Comment:
What's going on with the commented out code?

Reply:
a remnant of ancient times 😉